composer.json

```
@@ -19,14 +19,16 @@
   ],
   "require": {
     "php": "^8.1",
-    "api-platform/core": "^2.6.4",
-    "symfony/http-client": "^6.0",
-    "symfony/property-access": "^6.0",
-    "symfony/serializer": "^6.0",
-    "symfony/validator": "^6.0",
-    "symfony/event-dispatcher": "^6.0",
+    "api-platform/core": "^3",
```

```suggestion
    "api-platform/core": "^3.0",
```
To prevent an unexpected upgrade to 4.x